engine/execution/computation/execution_verification_test.go

```diff
@@ -160,7 +160,7 @@ func Test_ExecutionMatchesVerification(t *testing.T) {
 		err = testutil.SignTransaction(addKeyTx, accountAddress, accountPrivKey, 0)
 		require.NoError(t, err)
 
-		minimumStorage, err := cadence.NewUFix64("0.00008164")
+		minimumStorage, err := cadence.NewUFix64("0.00008312")
```

Comment:
The reason this needed a change is because `executeBlockAndVerify` also does bootstrapping and it is one of the bootstrapped accounts that dictates the minimum possible minimum account balance.